projects/CNSPRCY/services/cnsprcy/module.nix

```
@@ -0,0 +1,86 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.cnsprcy;
+in
+{
+  options.services.cnsprcy = {
+    enable = lib.mkEnableOption "cnsprcy";
+    package = lib.mkPackageOption pkgs "cnsprcy" { };
+
+    hostname = lib.mkOption {
+      type = lib.types.str;
+      description = "Hostname of CNSPRCY server";
+      default = builtins.getEnv "HOSTNAME";
```

I feel like [networking.hostName](https://search.nixos.org/options?channel=unstable&show=networking.hostName&from=0&size=50&sort=relevance&type=packages&query=networking.hostName) is more appropriate than reading an env var, here

```suggestion
      default = config.networking.hostName;
```